app/assets/javascripts/initializers/initializeBaseUserData.js

```diff
@@ -93,7 +93,7 @@ function addRelevantButtonsToComments(user) {
 }
 
 function toggleModActionMenu() {
-  document.querySelector('.mod-actions-menu').classList.toggle('hidden');
+  document.querySelector('.mod-actions-menu').classList.toggle('showing');
```

Comment:
I had to swap to `showing` since `hidden` was adding styles that were really hard to overwrite.